buildozer/README.md

```diff
@@ -0,0 +1,106 @@
+# Buildozer
+
+Buildozer is a command line tool to refactor multiple [Bazel](https://github.com/bazelbuild/bazel) BUILD files using standard commands. 
+
+### Dependencies
+
+1. Protobuf go runtime: to download  
+`go get -u github.com/golang/protobuf/{proto,protoc-gen-go}`
+
+
+### Installation
+
+1. Change directory to the buildifier/buildozer
+
+```bash
+gopath=$(go env GOPATH)
+cd $gopath/src/github.com/bazelbuild/buildifier/buildozer
+```
+
+2. Install
+```bash
+go install
+```
+
+### Usage
+`buildozer [OPTIONS] ['command args' | -f FILE ] label-list`
+
+Here, `label-list` is a comma separated list of Bazel labels, for example `//path/to/pkg1:rule1, //path/to/pkg2:rule2`. Buildozer reads commands from FILE ( - for stdin (format:|-separated command line arguments to buildozer, excluding flags))
+
+OPTIONS can be one of the following:
```

Comment:
I worry that options will be added without anyone updating the documentation and then this will become stale.

Perhaps mention the most useful commands and flags, and then "use buildifier -help for a complete listing"?